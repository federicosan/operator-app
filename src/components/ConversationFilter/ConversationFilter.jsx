import React, { Component, PropTypes } from 'react';
import moment from 'moment';
import { connect } from 'react-redux';

import { Spinner } from '@blueprintjs/core';

import { openConversation } from '../../store/Chat/actions';

export class ConversationFilterComponent extends Component {
  static propTypes = {

  }

  constructor (props) {
    super(props);

  }

  render() {
    const { dispatch } = this.props;

    return null;
  }
}

const mapStateToProps = state => ({

});

const mapDispatchToProps = dispatch => ({
  dispatch,
});

const ConversationFilter = connect(
  mapStateToProps,
  mapDispatchToProps,
)(ConversationFilterComponent);

export default ConversationFilter;
