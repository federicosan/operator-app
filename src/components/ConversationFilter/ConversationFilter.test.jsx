import React from 'react';

import { shallow } from 'enzyme';

import ConversationFilter from './ConversationFilter.jsx';

const store = {
  subscribe: jest.fn(),
  dispatch: jest.fn(),
  getState: jest.fn(() => ({ })),
};

describe('Chat', () => {
  it('matches snapshot', () => {
    const component = shallow(<ConversationFilter store={store} />);

    expect(component).toMatchSnapshot();
  });
});
