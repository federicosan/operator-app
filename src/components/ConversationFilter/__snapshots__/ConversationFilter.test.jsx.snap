exports[`Chat matches snapshot 1`] = `
<ConversationFilterComponent
  dispatch={[Function]}
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "subscribe": [Function],
    }
  } />
`;
